cmd/standalone/main.go

```diff
@@ -118,6 +119,17 @@ func startCollector(logger *zap.Logger, baseFactory metrics.Factory, memoryStore
 func startQuery(logger *zap.Logger, baseFactory metrics.Factory, memoryStore *memory.Store) {
 	metricsFactory := baseFactory.Namespace("jaeger-query", nil)
 
+	jaegerClientConfig.Configuration{
+		Sampler: &jaegerClientConfig.SamplerConfig{
+			Type:  "const",
+			Param: 1.0,
+		},
+		RPCMetrics: true,
+	}.InitGlobalTracer(
```

Comment:
my bad. We should change it to not use global tracer, but to expect tracer as an option.